src/pages/support/[secret].tsx

```diff
@@ -60,7 +61,11 @@ const support: NextPage<{ secret: string }> = (props): JSX.Element => {
         </div>
       </div>
       <div className="row">
-        <div className="col-12 col-md-6 mt-2">
+        <div className="col-12 mt-2">
+          <SupportRequests secret={props.secret} />
+        </div>
+
+        {/* <div className="col-12 col-md-6 mt-2">
```

Comment:
For the commented code, just delete it. If we're commenting it out, then we no longer need it. Also delete the files that this is referencing, as long as they aren't being used anywhere else. 👍